### ISSUE TITLE
Connecting with two servers to one agent

### ISSUE THREAD

**Issue by bravery-only** [Ylianst/MeshCentral#4869]
According to this photo, is it possible for the agent to be connected to two servers? If one server is disconnected, the agent can be connected to the second server.
![lMRZILIybS](https://user-images.githubusercontent.com/42680639/208682212-70783b3f-8503-4867-9bdd-8cf45174f24b.png)

**Comment by dxdemetriou** [comment]
> According to this photo, is it possible for the agent to be connected to two servers? If one server is disconnected, the agent can be connected to the second server. ![lMRZILIybS](https://user-images.githubusercontent.com/42680639/208682212-70783b3f-8503-4867-9bdd-8cf45174f24b.png)

https://github.com/Ylianst/MeshCentral/issues/3208

**Comment by dxdemetriou** [comment]
> According to this photo, is it possible for the agent to be connected to two servers? If one server is disconnected, the agent can be connected to the second server. ![lMRZILIybS](https://user-images.githubusercontent.com/42680639/208682212-70783b3f-8503-4867-9bdd-8cf45174f24b.png)

Please check here: Multiple servers for one agent #3208

**Comment by si458** [comment]
> Please check here: Multiple servers for one agent #3208

that issue is about running multiple instances of the meshagent to connnect to different servers, which is different from your issue

your issue is you want 1 instance to connect to multiple servers,
im not sure ive ever seen the image your provided before? where are you seeing that image?

**Comment by bravery-only** [comment]
I created this image myself. Two servers with one agent. If the connection with one agent fails, connect to the second server

**Comment by bravery-only** [comment]
> > According to this photo, is it possible for the agent to be connected to two servers? If one server is disconnected, the agent can be connected to the second server. ![lMRZILIybS](https://user-images.githubusercontent.com/42680639/208682212-70783b3f-8503-4867-9bdd-8cf45174f24b.png)
> 
> Please check here: Multiple servers for one agent #3208

I tried this method and I don't mean this method

**Comment by arduin0guru** [comment]
Hi, 
Is it possible for a single agent to connect to SERVER1, and if server1 is down, it connects to server2?
If SERVER1 is physically down, or if the internet service provider is down, you can still access it using SERVER2 in a different location or with a different service provider.

**Comment by si458** [comment]
@arduin0guru no, the only way to achieve this is 2 physical meshcentral servers with different customisations setup

`mycompant agent primary` and `mycompany agent backup` for example as machines can run multiple agents so longas they are customised differently
